csrc/device_lower/analysis/predicate_elimination.cpp

```diff
@@ -45,7 +45,8 @@ void assertOnWarpOps(const Expr* expr) {
     NVF_ERROR(in_tv != nullptr);
 
     NVF_ERROR(in_tv->definition() != nullptr);
-    bool is_tma_ldmatrix = ir_utils::isCpAsyncBulkLoad(in_tv->definition());
+    bool is_tma_ldmatrix = ir_utils::isCpAsyncBulkLoad(in_tv->definition()) &&
```

Comment:
```suggestion
    bool is_tma_load = ir_utils::isCpAsyncBulkLoad(in_tv->definition()) &&
```

`ldmatrix` is confusing with the actual `ldmatrix` ptx command.